docs/rules/order.md

```diff
@@ -89,3 +91,48 @@ You can set the options like this:
 ```js
 "import/order": ["error", {"groups": ["index", "sibling", "parent", "internal", "external", "builtin"]}]
 ```
+
+### `newlines-between: [always|never]`:
+
+
+Enforces or forbids new lines between import groups:
+
+- If omitted, assertion messages will be neither enforced nor forbidden.
+- If set to `always`, a new line between each group will be enforced, and new lines inside a group will be forbidden.
+- If set to `never`, no new lines are allowed in the entire import section.
+
+For default group setting those will be valid:
+
+```js
+/* eslint import/order: ["error", {"newlines-between": "always"}] */
+var fs = require('fs');
+
+var index = require('./');
+
+var sibling = require('./foo');
+```
+
+```js
+/* eslint import/order: ["error", {"newlines-between": "never"}] */
+var fs = require('fs');
+var index = require('./');
+var sibling = require('./foo');
+```
+
+while those will be invalid:
```

Comment:
`while the following will be invalid:`

Actually, if you could swap both sections, the invalid example is always (afaik) put before the valid section.